### PR TITLE
[Fix #9925]: fix ellipsis bug in table column

### LINKED ISSUE #9925: [Bug] [UI] Ellipsis in Table Column not work well
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

ellipsis in table column with `ButtonLink` not work well.

project-name

TaskInstance

### What you expected to happen

expect to have ellipsis in button link.
expect1

### How to reproduce

1. go to a page which have button link in table
2. edit it to a long length name

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rockfang**
## Purpose of the pull request
close [#9925 ](https://github.com/apache/dolphinscheduler/issues/9925)

## Brief change log
use an independent NEllipsis instead of  `ellipsis config in NDataTable`
## Verify this pull request
This pull request is code cleanup without any test coverage.

### CODE REVIEW

File: dolphinscheduler-ui/src/views/projects/list/use-table.ts [hunk 1]
```diff
@@ -76,23 +76,22 @@ export function useTable() {
         title: t('project.list.project_name'),
         key: 'name',
         className: 'project-name',
-        ...COLUMN_WIDTH_CONFIG['name'],
```
Comment (labbomb) [inline dolphinscheduler-ui/src/views/projects/list/use-table.ts:79]: you'd better not delete the content of config

File: dolphinscheduler-ui/src/views/projects/list/use-table.ts [hunk 2]
```diff
@@ -76,23 +76,22 @@ export function useTable() {
...
+        width: 200,
```
Comment (labbomb) [inline dolphinscheduler-ui/src/views/projects/list/use-table.ts]: There's no need to add this

File: dolphinscheduler-ui/src/views/projects/list/use-table.ts [hunk 3]
```diff
@@ -76,23 +76,22 @@ export function useTable() {
...
-                  {
-                    onClick: () => {
-                      router.push({ path: `/projects/${row.code}` })
-                    }
-                  },
-                  { default: () => row.name }
-                ),
-              tooltip: () => row.name
+                  NEllipsis,
+                  { style: 'max-width: 180px;line-height: 1.5' },
```
Comment (labbomb) [inline dolphinscheduler-ui/src/views/projects/list/use-table.ts]: Set width to the same as that specified in config, like: (max-width: 100px)

**Comment by labbomb** [comment]
thanks,I reviewed a file and gave an example, and other files were also modified by reference

**Comment by rockfang** [comment]
> thanks,I reviewed a file and gave an example, and other files were also modified by reference

hi bro, I didn't find your review and example.

**Comment by labbomb** [comment]
> > thanks,I reviewed a file and gave an example, and other files were also modified by reference
> 
> hi bro, I didn't find your review and example.

image

**Comment by rockfang** [comment]
@labbomb hi bro. i modifyid it.

**Review by labbomb — APPROVED** [review]
LGTM

**Review by songjianet — APPROVED** [review]
LGTM

**Pull request merged**